Client/src/Components/NavBar/index.jsx

```diff
@@ -51,6 +50,10 @@ function NavBar() {
   const navigate = useNavigate();
   const authState = useSelector((state) => state.auth);
 
+  let settings = ["Profile", "Password", "Team", "Logout"];
+  if (authState.user.role[0] !== "admin")
```

Comment:
It's not guaranteed that "admin" will be the first role in the `role` array, this should probably search the array to check if "admin" exists anywhere in the array